app/controllers/recipes_controller.rb

```diff
@@ -0,0 +1,56 @@
+# app/controllers/posts_controller.rb
+class RecipesController < ApplicationController
+  before_action :authenticate_user!
+
+  def index
+    @user = current_user
+    @recipes = Recipe.all
+  end
+
+  def show
+    @user = current_user
+    # @user = User.find(params[:user_id])
+    # @recipe = @user.recipes.includes(:user).find(params[:id])
```

Comment:
### [OPTIONAL]
Please remove the commented code to make your codebase look cleaner and professional 👍